driver/src/main/scala/api/MongoDriver.scala

```diff
@@ -200,6 +233,17 @@ class MongoDriver(
    *
    * $seeConnectDBTutorial
    *
+   * @param uri the URI to be parsed by [[reactivemongo.api.MongoConnection.parseURI]]
+   * @param name $connectionNameParam
+   * @param strictUri $strictUriParam
+   */
+  def askConnection(uri: String, name: Option[String], strictUri: Boolean): Future[MongoConnection] = Future.fromTry(MongoConnection.parseURI(uri)).flatMap(askConnection(_, name, strictUri))(system.dispatcher)
```

Comment:
Such connection not using strict URI should rather be kept `private` (or be inlined where needed), as at some point accepting URI with invalid option will no longer be accepted (no such not be part of new API)